src/main/java/mate/academy/spring/dto/request/MovieSessionRequestDto.java

```diff
@@ -1,10 +1,15 @@
 package mate.academy.spring.dto.request;
 
 import java.time.LocalDateTime;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
 
 public class MovieSessionRequestDto {
+    @Positive
```

Comment:
You should also use @NotNull .  @Positive does not prevent using null. Fix it in other places.